mod.config

```diff
@@ -89,3 +89,8 @@ PACKAGING_OSX_LAUNCHER_SOURCE="https://github.com/OpenRA/OpenRALauncherOSX/relea
 
 # Temporary file name used when downloading the OpenRA macOS launcher files.
 PACKAGING_OSX_LAUNCHER_TEMP_ARCHIVE_NAME="launcher.zip"
+
+# Enable the specified packaging method.
+PACKAGING_WINDOWS_ENABLED="True"
+PACKAGING_OSX_ENABLED="True"
+PACKAGING_ZIP_ENABLED="True"
```

Comment:
I have a strong philosophical objection to the idea of making it easy to exclude support for an OS by setting a config flag and then forgetting about it.